src/conf_mode/protocols_mpls.py

```diff
@@ -37,6 +37,14 @@ def get_config(config=None):
     mpls_conf = {
         'router_id'  : None,
         'mpls_ldp'   : False,
+        'old_parameters' : {
+                'no-ttl-propagation'          : False,
```

Comment:
Maybe "no_ttl_propagation" instead "no-ttl-propagation"

Reply:
Yes, you are correct my friend. I missed that. I'll see if I can make those changes in this PR.